### PR TITLE
Tamil translation for yakyak

### PR DESCRIPTION
**Pull request by Adithya-Ramanitharan**
I have translated the yakyak app's interface in Tamil language

### CODE REVIEW

**Comment by Adithya-Ramanitharan** [comment]
I am new to github . So can you please say how this system works ??

does the checks passed means my translation work will be added to the app in next release or it will have to wait for another expert's review

File: ta.json [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                "icon": "Show YakYak icon in notifications",
+                "custom_sound": "Use YakYak custom sound for notifications"
+            },
+            "color_scheme": {
+                "title": "நிற திட்டம்",
+                "default": "வழமையான நிறம்",
+                "blue": "நீலம்",
+                "dark": "கருப்பு",
+                "material": "Material"
+
```
Comment (averissimo) [inline ta.json]: please delete this empty line (140)

File: ta.json.template [hunk 2]
```diff
@@ -0,0 +1,174 @@
+{
```
Comment (averissimo) [inline ta.json.template]: please delete this file `ta.json.template` as it is a duplicate of the translation.

Only the `en.json.template` is necessary as it is works as the template for all translations

File: ta.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+            "undo": "மீளமை",
+            "redo": "மறுபடியும் செய்",
+            "paste": "ஒட்டு",
+            "cut": "வெட்டு",
+            "copy": "பிரதி செய்",
+            "copy_image_link": "படத்தின் link பிரதி செய்",
+            "copy_link": "link பிரதி செய்]",
+            "copy_image": "படத்தை பிரதி செய்",
+            "select_all": "அனைத்தையும் தெரிவு செய்",
+            "language": "மொழி (மொழி)"
```
Comment (averissimo) [inline ta.json]: the parenthesis should be in english, such as: 

"மொழி (Language)"

**Comment by averissimo** [comment]
@Adithya-Ramanitharan __thanks for your contribution!__

To answer your question the checks only guarantee that it passes the automatic tests we have in place.

I've added some minor comments for you to address, please see them above in this thread.

**Comment by Adithya-Ramanitharan** [comment]
I have made the changes.
What should I do now?

**Comment by Adithya-Ramanitharan** [comment]
I have made changes as you requested.

**Comment by averissimo** [comment]
Awesome!

image

**Comment by averissimo** [comment]
Some strings in the menus are not translated, are they untranslatable?

ps. this is me just being curious

**Comment by Adithya-Ramanitharan** [comment]
Thank you.
I will try to translate them too.

**Comment by Adithya-Ramanitharan** [comment]
Hello there,

I just signed the petition "END CORPORAL PUNISHMENT IN SCHOOLS IN SRI LANKA
- VISION 2020" and wanted to see if you could help by adding your name.

Our goal is to reach 1,223 signatures and we need more support. You can
read more and sign the petition here:

https://chn.ge/2OoGueo

Thanks!
Adithya

**Pull request merged**